dpc/src/base_dpc/outer_circuit/outer_circuit_gadget.rs

```diff
@@ -193,30 +193,6 @@ where
     let ledger_digest_fe = ToConstraintField::<C::InnerField>::to_field_elements(ledger_digest)
         .map_err(|_| SynthesisError::AssignmentMissing)?;
 
-    let mut serial_numbers_fe = Vec::with_capacity(old_serial_numbers.len());
```

Comment:
Can we revert these changes?

Changing this gadget will invalidate the current outer-circuit-pk and -vk parameters used by testnet1.

I will confirm this has been caught by CI

Reply:
As we skip DPC tests in CI, this compatibility break isn't being checked. Running locally now